src/ucp/proto/proto_debug.c

```diff
@@ -233,11 +232,8 @@ ucp_proto_select_elem_info(ucp_worker_h worker,
     do {
         range_start = range_end + 1;
 
-        proto_valid = ucp_proto_select_elem_query(worker, select_elem,
-                                                  range_start, &proto_attr);
-        if (!proto_valid) {
-            break;
-        }
+        ucp_proto_select_elem_query(worker, select_elem, range_start,
```

Comment:
IMO we should not print stub protocol ranges - if a range is missing we can understand it's a stub
on other hand, we reduce the size of the log ouput